open-machine-learning-jupyter-book/assignments/data-science/Soda-profits.ipynb

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
<p>Please also give credit to the data source creator? <a href="https://github.com/microsoft/Data-Science-For-Beginners/blob/main/2-Working-With-Data/06-non-relational/assignment.md" target="_blank">https://github.com/microsoft/Data-Science-For-Beginners/blob/main/2-Working-With-Data/06-non-relational/assignment.md</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>DONE</p>